staging/src/k8s.io/api/core/v1/generated.proto

```diff
@@ -2637,11 +2637,14 @@ message PersistentVolumeClaimSpec {
   // +optional
   optional string volumeMode = 6;
 
-  // This field requires the VolumeSnapshotDataSource alpha feature gate to be
-  // enabled and currently VolumeSnapshot is the only supported data source.
-  // If the provisioner can support VolumeSnapshot data source, it will create
-  // a new volume and data will be restored to the volume at the same time.
-  // If the provisioner does not support VolumeSnapshot data source, volume will
+  // This field can be used to specify either:
+  // * An existing VolumeSnapshot object (snapshot.storage.k8s.io/VolumeSnapshot - Beta)
+  // * An existing PVC (PersistentVolumeClaim - GA)
```

Comment:
You shouldn't have to say PVC is GA (since anything not called out as alpha or beta is implicitly GA).  Remove the `- GA` part (but keep `- Beta` on snapshot).